launchable/commands/subset.py

```diff
@@ -117,14 +117,26 @@ def separator(self, s: str):
             self._separator = s
 
         def test_path(self, path: TestPathLike):
+            def rel_base_path(path: str):
```

Comment:
test_runners sometimes call test_path but the test_path method hasn't considered the base option.
so I fixed it.

Let me make sure, just in case. 
we should trim the base path when the user set base option, is this right?